src/helix-ui/styles/elements/hx-tile.less

```diff
@@ -7,15 +7,23 @@ hx-tile {
   text-align: center;
   width: 100%;
 
-  hx-tile-title {
```

Comment:
Please don't remove the old CSS just yet. We want the deprecation to be backward-compatible.

Make sure to move/copy the old definitions to a section at the bottom of this file for deprecation.